assets/l10n/app_en.arb

```diff
@@ -456,9 +456,9 @@
   "@recentDmConversationsPageTitle": {
     "description": "Title for the page of recent DM conversations"
   },
-  "allMessagesPageTitle": "All messages",
+  "allMessagesPageTitle": "Combined feed",
   "@allMessagesPageTitle": {
-    "description": "Title for the page of all messages"
+    "description": "Title for the page of combined feed."
   },
```

Comment:
nit: let's call it `combinedFeedPageTitle` instead of `allMessagesPageTitle`

Reply:
Done 👍